config/.env

```diff
@@ -1,8 +1,4 @@
 PORT=3000
-DB_NAME=skynet
-DB_USER=
-DB_PASSWORD=
-DB_HOST=localhost
-DB_PORT=5432
+DATABASE_URL=postgres://<DB_USER_HERE>:<DB_PASSWORD_HERE>@localhost:5432/skynet
```

Comment:
I actually don't like using this URL. Where we need the URL, we can have a private method or interpolated string that uses up the DB_NAME, DB_PASSWORD, HOST, PORT, etc. Having it all in one place makes it less configurable

Reply:
I think we have to use the URL string because Heroku periodically changes the database credentials and automatically sets/updates them in the DATABASE_URL config var

![image](https://user-images.githubusercontent.com/48368341/67147287-9231d680-f261-11e9-9996-e3d753adc5f2.png)